src/main/java/org/mskcc/oncokb/curation/service/FirebaseService.java

```diff
@@ -0,0 +1,93 @@
+package org.mskcc.oncokb.curation.service;
+
+import com.google.firebase.database.*;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.mskcc.oncokb.curation.domain.Alteration;
+import org.mskcc.oncokb.curation.domain.Association;
+import org.mskcc.oncokb.curation.domain.Evidence;
+import org.mskcc.oncokb.curation.domain.enumeration.EvidenceType;
+import org.mskcc.oncokb.curation.domain.firebase.Gene;
+import org.mskcc.oncokb.curation.domain.firebase.Mutation;
+import org.mskcc.oncokb.curation.domain.firebase.MutationEffect;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class FirebaseService {
+
+    private final Logger log = LoggerFactory.getLogger(FirebaseService.class);
+    private final GeneService geneService;
+    private final AlterationService alterationService;
+    private final EvidenceService evidenceService;
+    private final AssociationService associationService;
+    private FirebaseDatabase database;
+
+    public FirebaseService(
+        GeneService geneService,
+        AlterationService alterationService,
+        EvidenceService evidenceService,
+        AssociationService associationService
+    ) {
+        this.geneService = geneService;
+        this.alterationService = alterationService;
+        this.evidenceService = evidenceService;
+        this.associationService = associationService;
+    }
+
+    public void readGene() {
+        String PATH = "Genes/BRAF";
```

Comment:
I see that FirebaseService is not used at the moment, but its supposed to get the gene data from FB and convert to FB Java models so that we can use it based on my understanding.

Should the gene name be a param that is passed?

Reply:
This is just for demo purpose. Not a complete list, I will send a separate PR to finish the importing work.